### PR TITLE
CID-15512: Help center - 'Manage NHS Record - NHS record contact details' article update

### PR DESCRIPTION
**Pull request by umair-nhs**
[CID-15512](https://nhsd-jira.digital.nhs.uk/browse/CID-15512)

### CODE REVIEW

File: src/manage/nhs-record-contact-details.md [hunk 1]
```diff
@@ -8,26 +8,28 @@ position: 6
...
 
 ---
 
-## Your NHS login contact details are different to the contact details on your NHS record
-
-The contact details on your NHS login may be different to those on your NHS record. If this is the case, we may ask you if you would like to update them on your NHS record.
+## How to add or update your NHS contact details
 
-We will show you the contact details we have for you. For example, we will show you the phone number we have from your NHS login and phone numbers on your NHS record. You can then choose which one the NHS should use to contact you. We will update your NHS record with this phone number. After this, the other phone numbers will not be in use on your NHS record.
+You can add or update the NHS contact details on your NHS record by [logging into your NHS account](https://access.login.nhs.uk/enter-email 'logging into your NHS account') or by [using the NHS App](https://www.nhsapp.service.nhs.uk/login 'by using the NHS App').
```
Comment (lps88) [inline src/manage/nhs-record-contact-details.md:23]: It's not a great idea to directly link to enter email page, it won't work if you don't have any session initialised. You could put a link for settings.login.nhs.uk and then they'd be sent to enter email if they're not already logged in

File: src/manage/nhs-record-contact-details.md [hunk 2]
```diff
@@ -8,26 +8,28 @@ position: 6
...
 
 ---
 
-## Your NHS login contact details are different to the contact details on your NHS record
-
-The contact details on your NHS login may be different to those on your NHS record. If this is the case, we may ask you if you would like to update them on your NHS record.
+## How to add or update your NHS contact details
 
-We will show you the contact details we have for you. For example, we will show you the phone number we have from your NHS login and phone numbers on your NHS record. You can then choose which one the NHS should use to contact you. We will update your NHS record with this phone number. After this, the other phone numbers will not be in use on your NHS record.
+You can add or update the NHS contact details on your NHS record by [logging into your NHS account](https://settings.login.nhs.uk 'logging into your NHS account') or by [using the NHS App](https://www.nhsapp.service.nhs.uk/login 'by using the NHS App').
```
Comment (lps88) [inline src/manage/nhs-record-contact-details.md:23]: super picky but `[using the NHS App]` is not matching `'by using the NHS App'`

File: src/manage/nhs-record-contact-details.md [hunk 3]
```diff
@@ -8,26 +8,28 @@ position: 6
...
 
-If some contact details are missing from your NHS record, we will ask if you would like to add the details from your NHS login. For example, if your NHS record does not have an email address, we will ask if you would like to update it with the email address you use for NHS login.
+<div class="nhsuk-warning-callout">
+    <h3 class="nhsuk-warning-callout__label">
```
Comment (lps88) [inline src/manage/nhs-record-contact-details.md:26]: I just span this up locally and there's a bit of a weird spacing thing on the callout label, I'm not sure that there is a neater way to fix it than just sticking the style on this element `style="padding-top: 8px;"` or maybe you could add a new stylesheet in

File: src/manage/nhs-record-contact-details.md [hunk 4]
```diff
@@ -8,26 +8,28 @@ position: 6
...
+    <span role="text">
+        <span class="nhsuk-u-visually-hidden">Important</span>
```
Comment (lps88) [inline src/manage/nhs-record-contact-details.md:28]: is there any need for this hidden text? I think screenreaders will just read this out twice now :smile:

**Review by lps88 — APPROVED** [review]

**Review by athavansri-nhs — APPROVED** [review]

**Pull request merged**